src/Contract.php

```diff
@@ -72,19 +79,26 @@ public function toPdf(HtmlConverterInterface $htmlConverter, array $data = [])
     /**
      * Generates file and sends a SignRequest
      * @param DocumentSignInterface $documentSign
+     * @return bool|void
      */
     public function sign(DocumentSignInterface $documentSign)
     {
-        if (!$this->file || !config('mvc-contracts.document_sign.enabled')) {
+        $config = config('mvc-contracts.document_sign');
+
+        if (!$this->file || !Arr::get($config, 'enabled', '')) {
             return;
         }
 
         try {
             $document = $documentSign->create(storage_path($this->file), Str::replaceFirst('contracts/', '', $this->file));
             $this->external_id = $document['id'];
-            $documentSign->signer($this->user->email)->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
-            $this->save();
-            return true;
+
+            $redirectUrl = '';
+            if (Arr::has($config, 'return')) {
+                $redirectUrl = str_replace('{id}', $this->id, Arr::get($config, 'return', ''));
```

Comment:
Arr::get is niet meer nodig want je hebt al een Arr::has gedaan en gebruik een short if statement